doc/source/usage.rst

```diff
@@ -0,0 +1,194 @@
+.. _usage:
+
+Usage
+=====
+
+Overview
+--------
+
+This sections dives deeper into the inner workings of **DataStructureDefinition** and
+**RegionProcessor** as well as their associated auxiliary classes.
+
+
+Yaml file schemas
+--------------------
+
+Both classes read information from yaml files which are required to follow specific
+structures.
+
+For **DataStructureDefinition** there are three types of schemas: variable, region and tag.
```

Comment:
Suggest to add a TOC at the top of this document and use the class names as section heading

Reply:
Added a toc